apps/web/src/design-system/table/Table.tsx

```diff
@@ -144,7 +144,7 @@ export function Table({
           })}
         </tbody>
       </MantineTable>
-      {pagination && total && pageSize > 1 && (
+      {pagination && total > 0 && pageSize > 1 && (
```

Comment:
:o Actually don't get why this was happening in the first place? Why is this was evaluated as "0", because the last block was a react component?

Reply:
yeah and when I do `{typeof total}` on the line above it says `number` so...

Reply:
I think it is because pagination is not null, it is an object with total = 0, called from the activity page

Reply:
Yeah @ainouzgali ! That's called short circuit evaluation in javascript https://codeburst.io/javascript-what-is-short-circuit-evaluation-ff22b2f5608c

Never seen it behaves like this, I was thinking it will evaluate to false and react won't render this.